app/main.py

```diff
@@ -1,6 +1,33 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+
+load_dotenv()
+API_KEY = os.getenv("API_KEY")
+BASE_URL = "https://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    request = (
+        requests
+        .get(
+            f"{BASE_URL}/current.json?q={FILTERING}&key={API_KEY}"
+        ).json())
+
+    location = request["location"]
+    current = request["current"]
+
+    print(
+        f"{location["name"]}/"
+        f"{location["country"]} "
+        f"{location["localtime"]} "
+        f"Weather: {current["temp_c"]} Celsius "
+        f"{current["condition"]["text"]}"
```

Comment:
The print statement uses double quotes inside an f-string, which will cause a syntax error. You should use single quotes for the keys inside the f-string to avoid this issue. For example, use `f"{location['name']}"` instead of `f"{location["name"]}"`.